### PR TITLE
feat: add terms of service confirmation to signup

### PR DESCRIPTION
**Pull request by 12ian34**

### CODE REVIEW

File: packages/frontend/src/pages/Signup.tsx [hunk 1]
```diff
@@ -213,6 +213,13 @@ const Signup: FC = () => {
                             >
                                 Privacy Policy.
                             </FooterCta>
+                            and our{' '}
+                            <FooterCta
```
Comment (ZeRego) [inline packages/frontend/src/pages/Signup.tsx:217]: We also need this in `packages/frontend/src/pages/Register.tsx`

**Review by ZeRego — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/Signup.tsx (hunk 1 above)
Comment (12ian34) [reply packages/frontend/src/pages/Signup.tsx:217]: Added!

**Comment by ZeRego** [comment]
@12ian34 
There is a missing space:
Captura de ecrã 2023-02-24, às 11 10 56

We could also try to add a line break so the text feels more balanced. What do you think? 
Captura de ecrã 2023-02-24, às 11 12 33

**Comment by 12ian34** [comment]
> There is a missing space
darn, ok!

> We could also try to add a line break so the text feels more balanced. What do you think?
sounds good - would this just be a `\n` or do I need to actually add a new-line? @ZeRego

**Comment by ZeRego** [comment]
> sounds good - would this just be a \n or do I need to actually add a new-line? @ZeRego

no, you need to use html instead -> `<br/>`

**Review by ZeRego — APPROVED** [review]

**Pull request merged**